### DISCUSSION TITLE
Federation logo and feed

### DISCUSSION THREAD

**Discussion by edmael** [owncast/owncast#1960] [Q&A] [answered]
Hey, I've got a couple of questions on how Owncast federation works.

- I've changed my logo on the admin panel but it keeps showing up with the old one on my federated profile.
- Is there a way to remove the post published on my profile?

Thanks!

**Comment by gabek** [comment, marked as answer]
That logo thing is a known issue,Ffediverse servers are caching your logo and not refreshing it unfortunately. The next release of Owncast will force remote servers to re-fetch it with some cachebusting.

You currently cannot delete a post that you made, it's not really built to have a timeline or a microblogging scenario. In the future it's possible more robust writing and editing features will get built if that's something that's important to people. But not everyone wants or cares about Fediverse functionality, and I don't want to turn it into a microblogging platform as that would severely distract from the focus of the software. Right now Fediverse functionality is a value-add, but not the focus.

**Reply by edmael** [reply]
Thanks for your answer: I get the fact that the Federation is just a "plus", will wait for the next release for the logo issue ✌️
